### PR TITLE
[importer_direct_upload_blog] blog post for direct upload in importer is added

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- blog post for importer direct upload

### CODE REVIEW

File: docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+### Intermediate steps(ss):
+![Importer direct upload step1](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_step1.png)
+</br>
+  Selection of file in (small local file), Format guess and preview of table.
+</br>
+
+![Importer direct upload step2](https://cdn.gethue.com/uploads/2021/05/importer_direct_upload_step2.png)
+</br>
+  Selection of dialect and auto field type guesses. You can edit the name and type of fields by clicking at the Fields section.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:82]: Suggested edit: You select a SQL dialect and Hue auto-detects the column data types. You can edit column names and their data type.

File: docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - default
+categories:
+  - Version 4.10
+  - Development
+
+---
+
+If you’ve ever struggled with creating new SQL tables from CSV files, you’ll be happy to learn that this is now much easier.
+
+If you know the history of Hue then you must be familiar with the importer option in Hue. So basically earlier we could create a table from a file in different databases but there is one necessary condition, the file must be present in the filesystem storage like [HDFS](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction) or [s3](https://gethue.com/introducing-s3-support-in-hue/). Now we can create a table from local storage files also in different [databases](https://docs.gethue.com/administrator/configuration/connectors/).
```
Comment (romainr) [inline docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:49]: FYI: s3 --> S3

Comment (romainr) [reply docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:49]: 1.2 is super old, please link to the last one, e.g. https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html

File: docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+### Goal:
```
Comment (romainr) [inline docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:51]: Remove all `:` in titles?

File: docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: Importer direct upload
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:2]: Suggest rewording to something like "Improved Hue Importer -- Select a file, choose a dialect, create a table".

**Review by romainr — COMMENTED** [review]
Nice! 

A few small tweaks

File: docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+### Why:
+
+* If someone doesn't have HDFS or s3 configured in Hue then it is difficult to create the table from a file, so if we have the feature like this then anyone can use this feature for creation of tables. And ofcourse, it also saves time as it directly uploads the file to the importer.
+
+### Steps to create a table:
+
+![Importer direct upload steps gif](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_steps.gif)
+
+### Describing the steps through the app diagram:
+![Importer direct upload steps app diagram](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_app_diagram.png)
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md:64]: The blocks in the diagram look a bit inconsistent.
Could you please redo it? Suggestion:
Select a file --> Select a dialect (Hive, Impala, Phoenix, MySQL, ...) --> Review/Edit column names and data type --> Submit

Put the text in the boxes and keep the arrows blank.

**Review by pychheda — CHANGES_REQUESTED** [review]
Hi Ayush,
Thanks for running this by me.

I have left comments and provided some suggestions.
Hope you find this helpful.
Best,
Priyanka

**Review by romainr — APPROVED** [review]

**Comment by pychheda** [comment]
Looks great! Thanks.

File: docs/gethue/content/en/posts/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+Now it’s time to play with this feature in the latest Hue or at [demo.gethue.com](https://demo.gethue.com/hue/indexer/importer).  
+</br>
+</br>
+This project gladly welcome [contributions](https://github.com/cloudera/hue/#development) for supporting more SQL dialects.  
+Any feedback or question? Feel free to comment here or on the [Forum](https://discourse.gethue.com/) and [quick start](https://docs.gethue.com/quickstart/) SQL querying!
+
+Onwards!
+
+Ayush from the Hue Team
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table.md:95]: LGTM.

**Review by pychheda — APPROVED** [review]
